src/style.css

```diff
@@ -0,0 +1,99 @@
+@import url("./styles/star.css");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  line-height: 18px;
+  color: #060b35;
+
+  --color-blue: #00acdc;
+}
+
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin-top: 0;
```

Comment:
you can reset margins for tag and then apply only one

Reply:
reset default margin at `p` tag:
```
p {
  margin: 0
}
```